_es/tour/basics.md

```diff
@@ -6,4 +6,319 @@ discourse: false
+```
+
+Noten como la declaración de tipo `Int` va después del identificador `x`. También necesitas un `:`.
+
+## Variables
+
+Una variable es como un valor, excepto que a una variable se le puede re-asignar un valor después de declararla. Una variable se declara con la palabra reservada `var`.
+
+```tut
+var x = 1 + 1
+x = 3 // This compiles because "x" is declared with the "var" keyword.
+println(x * x) // 9
+```
+
+Como con los valores, si se quiere se puede especificar el tipo de una variable mutable:
+
+```tut
+var x: Int = 1 + 1
+```
+
+## Bloques
+
+Se pueden combinar expresiones rodeándolas con `{}` . A esto le llamamos un bloque.
+
+El resultado de la ultima expresión del bloque es también el resultado total del bloque.
+
+```tut
+println({
+  val x = 1 + 1
+  x + 1
+}) // 3
+```
+
+## Funciones
+
+Una función es una expresión que acepta parámetros.
+
+Una función se puede declarar anónima, sin nombre. Por ejemplo, ésta es una función que acepta un número entero `x`, y devuelve el resultado de incrementarlo `x+1`:
+
+```tut
+(x: Int) => x + 1
+```
+
+La lista de parámetros de la función está a la izquierda de la flecha `=>`, y a su derecha está el cuerpo de la función.
+
+También podemos asignarle un nombre a la función.
+
+{% scalafiddle %}
+```tut
+val addOne = (x: Int) => x + 1
+println(addOne(1)) // 2
+```
+{% endscalafiddle %}
+
+Las funciones pueden tomar muchos parámetros.
+
+{% scalafiddle %}
+```tut
+val add = (x: Int, y: Int) => x + y
+println(add(1, 2)) // 3
+```
+{% endscalafiddle %}
+
+O ninguno.
+
+```tut
+val getTheAnswer = () => 42
+println(getTheAnswer()) // 42
+```
+
+## Métodos
+
+Los métodos se parecen y comportan casi como a las funciones, pero se diferencian en dos aspectos clave: 
+
+Un método se define con la palabra reservada `def`, seguida por el nombre del método, la lista de parámetros, el tipo de valores que el método devuelve, y el cuerpo del método.
+
+{% scalafiddle %}
+```tut
+def add(x: Int, y: Int): Int = x + y
+println(add(1, 2)) // 3
+```
+{% endscalafiddle %}
+
+Observe que el tipo de retorno se declara _después_ de la lista de parámetros, y separado con dos puntos, e.g. `: Int`.
+
+Un método puede tener varias listas de parámetros
+
+{% scalafiddle %}
+```tut
+def addThenMultiply(x: Int, y: Int)(multiplier: Int): Int = (x + y) * multiplier
+println(addThenMultiply(1, 2)(3)) // 9
+```
+{% endscalafiddle %}
+
+O ninguna lista de parametros.
+
+```tut
+def name: String = System.getProperty("user.name")
+println("Hello, " + name + "!")
+```
+
+Hay otras diferencias, pero para simplificar, podemos pensar que son similares a las funciones.
+
+Los métodos también pueden tener expresiones de varias lineas.
+
+```tut
+def getSquareString(input: Double): String = {
+  val square = input * input
+  square.toString
+}
+```
+
+La ultima expresión en el cuerpo del método es el valor de retorno del mismo.
+(Scala tiene una palabra reservada `return`, pero se usa raramente, y no se aconseja usarla)
```

Comment:
```suggestion
(Scala tiene una palabra reservada `return`, pero se usa raramente y no se aconseja usarla)
```
I never heard or used the Oxford Comma in Castilian S panish. Not sure what's its adoption in other flavors od Spanish.